models/user.rb

```diff
@@ -137,4 +147,58 @@ def send_reset_password_email
 
     Emailer.mail(self.email, 'Nebraska Innovation Studio Password Reset', body)
   end
+
+  def notify_trainer_of_new_event(event)
+body = <<EMAIL
+<p>Hi, #{self.full_name}. You have been assigned as a trainer for <strong>#{event.title}</strong>. Don't forget that this event is</p>
+
+<p><strong>#{event.start_time.in_time_zone.strftime('%A, %B %d at %l:%M %P')}</strong>.</p>
+
+<p>We'll see you there!</p>
+
+<p>Nebraska Innovation Studio</p>
+EMAIL
+
+    Emailer.mail(self.email, "Nebraska Innovation Studio - Assigned as Trainer for #{event.title}", body)
+  end
+
+  def notify_trainer_of_modified_event(event)
+body = <<EMAIL
+<p>Hi, #{self.full_name}. You are receiving this email because an event you are training has been modified:</p>
+
+<p><strong>#{event.title}</strong></p>
+
+<p>You can see the event details on your <a href="https://innovationstudio-manager.unl.edu/home/">Home Page</a>.</p>
```

Comment:
You can make this link dynamic so that it works for the dev and production environments. If you look at my closed pull request for the promo email updates I did something like this for reference.